### ISSUE TITLE
Incorporate unit tests

### ISSUE THREAD

**Issue by philiporlando** [philiporlando/fgdb_to_gpkg#1]
Add unit tests using `pytest`.  Learn more about dev environments and including mock data within the package for testing.

**Comment by philiporlando** [comment]
The `fiona` package doesn't support writing to FileGDB, only reading... Need to figure out the best way to handle unit tests without attaching data to this package.

**Comment by philiporlando** [comment]
GDAL >=3.6 has a `OpenFileGDB` driver that should support writing to `.gdb`. Going to add the `ubuntugis-unstable` repo and try using this instead.

```
pipenv install --dev --pre --install-option="--gdal-config=/usr/bin/gdal-config" fiona
```

**Comment by philiporlando** [comment]
Still seeing errors with fiona:

```
fiona.errors.DriverError: OpenFileGDB driver requires at least GDAL 3.6.0 for mode 'w', Fiona was compiled against: 3.5.3
```

```
gdal-config --version
3.6.2
```

**Comment by philiporlando** [comment]
https://github.com/Toblerity/Fiona/issues/565

**Comment by philiporlando** [comment]
Filed a [new issue](https://github.com/Toblerity/Fiona/issues/1231) to track progress on fiona compliation against newer versions of GDAL.

**Comment by philiporlando** [comment]
This was resolved with 8b966575157f4c9947cc60319f7988495202241f
